modules/tensor_mechanics/test/tests/creep_tangent_operator/tests

```diff
@@ -0,0 +1,47 @@
+[Tests]
+  [./ten]
+    type = 'CSVDiff'
+    input = 'creep.i'
+    csvdiff = 'ten.csv'
+    cli_args = 'Materials/elastic_strain/inelastic_models="creep_ten" Outputs/file_base="ten"'
+  [../]
+  [./zero]
+    type = 'CSVDiff'
+    input = 'creep.i'
+    csvdiff = 'zero.csv'
+    cli_args = 'Materials/elastic_strain/inelastic_models="creep_ten creep_zero" Outputs/file_base="zero"'
+  [../]
+  [./sum]
+    type = 'CSVDiff'
+    input = 'creep.i'
+    csvdiff = 'sum.csv'
+    cli_args = 'Materials/elastic_strain/inelastic_models="creep_nine creep_one" Outputs/file_base="sum"'
+  [../]
+  [./cycle]
+    type = 'CSVDiff'
+    input = 'creep.i'
+    csvdiff = 'cycle.csv'
+    cli_args = 'Materials/elastic_strain/inelastic_models="creep_ten creep_ten2" Materials/elastic_strain/cycle_models=true Outputs/file_base="cycle"'
+  [../]
+
+  [./ten_jacobian]
+    type = 'AnalyzeJacobian'
+    input = 'creep.i'
+    cli_args = 'Materials/elastic_strain/inelastic_models="creep_ten" Outputs/file_base="ten"'
```

Comment:
Yeah, this is the idea!  One thing though, for tests that write out to the same file, you need to use the `prereq` option (`prereq = ten`) so you don't have two tests running in parallel both writing to the same file. That forces the prerequisite test to complete first before the one that depends on it is started. Other than fixing that up, I think this is good to go.

Reply:
Is that the case since I am changing the file_base?

Reply:
I thought it was either-or

Reply:
Or perhaps you mean I can just have a single gold file if I use prereqs?